sympy/stats/crv_types.py

```diff
@@ -1205,6 +1210,13 @@ def FDistribution(name, d1, d2):
 class FisherZDistribution(SingleContinuousDistribution):
     _argnames = ('d1', 'd2')
 
+    set = Interval(-oo, oo)
```

Comment:
Should this be `Interval(0, oo)`?

Reply:
The wiki link: https://en.wikipedia.org/wiki/Fisher's_z-distribution
says that `set = Interval(-oo, oo)`

Reply:
Ah, looking it up directed to "Fisher Distribution" https://en.wikipedia.org/wiki/F-distribution.